### ISSUE TITLE
Documentation for BTT Mini12864 display

### ISSUE THREAD

**Issue by kad** [bigtreetech/BIGTREETECH-TouchScreenHardware#6]
It would be good to get documentation for Mini12864 display.
Minimum needed is pinout and corresponding Marlin settings for it.

**Comment by thisiskeithb** [comment]
You can find details in the “Expansion Module” repo: https://github.com/bigtreetech/BTT-Expansion-module/tree/master/BTT%20Mini12864

**Comment by kad** [comment]
Yeah, support responded to same question privately back in January, however it would be good if they would be responding also publicly. And even in BTT-Expansion-module, the doc files can be improved, e.g. with schematics or at least pin-out information. Otherwise users need to fetch pinout information from fysetc wiki.
